src/ButtonStrip.js

```diff
@@ -0,0 +1,58 @@
+import React from 'react'
+import propTypes from '@dhis2/prop-types'
+import cx from 'classnames'
+
+import { Button } from './Button.js'
+import { spacers } from './theme.js'
+
+/**
+ * @module
+ * @param {ButtonStrip.PropTypes} props
+ * @returns {React.Component}
+ * @example import { ButtonStrip } from @dhis2/ui-core
+ * @see Live demo: {@link /demo/?path=/story/buttonstrip--default|Storybook}
+ */
+const ButtonStrip = ({ className, children, middle, end }) => (
+    <div className={cx(className, { middle, end })}>
+        {children}
+
+        <style jsx>{`
+            div {
+                display: flex;
+            }
+            div.middle {
+                justify-content: center;
+            }
+            div.end {
+                justify-content: flex-end;
+            }
+            div > :global(button) {
+                margin-left: ${spacers.dp16};
+            }
+        `}</style>
+    </div>
+)
+
+const alignmentPropType = propTypes.mutuallyExclusive(
+    ['middle', 'end'],
+    propTypes.bool
+)
+
+/**
+ * @typedef {Object} PropTypes
+ * @static
+ *
+ * @prop {string} [className]
+ * @prop {Array.<Button>} [children]
+ * @prop {boolean} [middle] - The props `middle`, and `end` are
+ * mutually exlusive
+ * @prop {boolean} [end]
+ */
+ButtonStrip.propTypes = {
+    className: propTypes.string,
+    children: propTypes.arrayOf(propTypes.instanceOfComponent(Button)),
+    middle: alignmentPropType,
```

Comment:
So maybe technically not a breaking change since `start` has become the default, but the `start` prop has been dropped.